### DISCUSSION TITLE
Negative electrode potentials strange behaviour even for standard parameter sets

### DISCUSSION THREAD

**Discussion by ss711** [pybamm-team/PyBaMM#3470] [General]
Hi when I look at the negative potentials for my own and even standard parameter sets the negative electrode potentials are strange in shape and are extremely low (10^-5) but the positive electrode potential makes sense. What is the reason for this?

See below for standard 1C discharge of Chen2020 parameter set:

![image](https://github.com/pybamm-team/PyBaMM/assets/64744013/aa59d067-14cc-448a-9d64-053d55ee1f92)

My code:
` 
import pybamm
from Chen2020 import parameter_values  

hrs = 10
dt = 10
sec_h = 3600
secs = hrs * sec_h
pts = int(secs/dt)
t_eval = np.linspace(0, secs, pts)
dt_max = 1

Crates = [1]

for i in Crates:

    model = pybamm.lithium_ion.DFN()  
    return_solution_if_failed_early=True
    
    sim = pybamm.Simulation(model, parameter_values=parameter_values, solver=pybamm.CasadiSolver(dt_max=dt_max),C_rate = i)
    sim.solve(t_eval)
    solution = sim.solution `

Thank you for your help

**Comment by rtimms** [comment]
We set the reference potential to be zero at the interface between the negative current collector and electrode, which is why the negative electrode potential is always close to zero. You can look at the OCPs of each electrode by looking at the variables "Negative electrode open circuit potential [V]" and "Positive electrode open circuit potential [V]". You can also look at the potential difference between the electrolyte and electrode at the separator interface by looking at the variables "Negative electrode surface potential difference at separator interface [V]" and "Positive electrode surface potential difference at separator interface [V]", which might be of interest too. FYI you can do model.variables.search("string") to search for variables.

See https://github.com/pybamm-team/PyBaMM/discussions/3254

**Reply by ss711** [reply]
Thank you Robert this is very helpful. Actually those "potential difference at separator interface" seem to be what I'm looking for as looking to see at high rate whether it is overpotentials at the negative or positive electrode which is causing the cutoff voltage to be reached. Do you know how I could see this?

Also can I ask how are the DFN electrode overpotentials assigned to either reaction overpotential vs electrode concentration overpotential? Many thanks for your help

**Reply by rtimms** [reply]
Take a look at the variables "Negative electrode surface potential difference at separator interface [V]" and "Positive electrode surface potential difference at separator interface [V]". You can also search variables to get the different overpotentials. Or use the `plot_voltage_components` function.

For how the overpotentials are broken down, see e.g. equation 48 in https://arxiv.org/pdf/1905.12553.pdf.
